README.md

```diff
@@ -28,6 +33,9 @@ npm run get-cards
 # rm -r build/  # needed if files are added / deleted / renamed between builds
 npm run build
 
+# start Gamenode server
+node ./build/server/gamenode
```

Comment:
I feel like if we add something about gamenode to the readme we should wait until we can give some complete instructions for a working example. Just this line would probably be kind of confusing since it's not clear what's supposed to happen or what gamenode is